CHANGELOG.md

```diff
@@ -6,7 +6,19 @@ Our versioning scheme is `YEAR.N` where `N` is incremented whenever a new releas
 
 ## [upcoming release]
 
-- Update garage-push and garage-deploy tools to support the latest backend changes. Both are backward compatible. Previous versions have the server URL *without* the token path, so it needs to be hardcoded. The new version has the full URL with the */oauth2/token* path at the end. Also, treehub.json has an additional parameter *scope*: [PR](https://github.com/advancedtelematic/aktualizr/pull/1767)
+## [2020.10] - 2020-10-27
+
+### Added
+- Updated garage-push and garage-deploy tools support new back-end token generation to authenticate API requests. Update treehub.json format evolved for the new back-end Both are backward compatible. Previous versions have the server URL *without* the token path, so it needs to be hardcoded. The new version has the full URL with the */oauth2/token* path at the end. Also, treehub.json has an additional parameter *scope*: [PR](https://github.com/advancedtelematic/aktualizr/pull/1767)
+
+### Changed
+- dockerfile use default ostree package: [PR](https://github.com/advancedtelematic/aktualizr/pull/1751)
+- Improve libaktualizr API exceptions: [PR](https://github.com/advancedtelematic/aktualizr/pull/1754)
+- Better logging for binary file download progress: [PR](https://github.com/advancedtelematic/aktualizr/pull/1756)
+- Fixed issue with parameters check in aktualizr-get: [PR](https://github.com/advancedtelematic/aktualizr/pull/1760)
+- Allow passing HTTP headers in aktualizr-get: [PR](https://github.com/advancedtelematic/aktualizr/pull/1762)
+- aktualizr-lite moved into its own [aktualizr-lite repo](https://github.com/foundriesio/aktualizr-lite): [PR](https://github.com/advancedtelematic/aktualizr/pull/1763)
+- Fixed output of pacman config: [PR](https://github.com/advancedtelematic/aktualizr/pull/1761)
```

Comment:
I would put the two items that start with "Fixed" into a separate *Fixed* subsection like in previous releases.